DESCRIPTION

```diff
@@ -27,7 +27,7 @@ Encoding: UTF-8
 Language: en-US
 LazyData: false
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.2.1
```

Comment:
@DeclanHodges this should not be changing.  We need to use the roxygen in the lock file